choose-an-avatar.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Henny+Penny|Montserrat+Alternates">
+  <link rel="stylesheet" href="node_modules/materialize-css/bin/materialize.css" media="screen,projection"/>
+  <link rel="stylesheet" href="main.css">
+
+  <title>Welcome to CAMHS</title>
+</head>
+<body>
+  <div class="avatar">
+    <h1 class="avatar__title">Choose an animal</h1>
+    <div class="carousel">
+      <a class="carousel-item" href="#one!"><img src="assets/lion.svg"></a>
+      <a class="carousel-item" href="#two!"><img src="assets/lion.svg"></a>
+      <a class="carousel-item" href="#three!"><img src="assets/lion.svg"></a>
+    </div>
+  </div>
+
+  <script src="node_modules/jquery/dist/jquery.min.js"></script>
+  <script src="node_modules/materialize-css/bin/materialize.js"></script>
```

Comment:
Using npm for your deps is cool. But if you want to build static site you will either need to commit these files to master or (better) write a build script that pulls these files out of node modules and saves them somewhere like `/assets`, then you could run it and commit these files under a more normal path to a gh-pages branch...
Does that make sense?